services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -57,7 +65,8 @@ async def docker_compose_config(
     [SEE docker-compose](https://docs.docker.com/engine/reference/commandline/compose_convert/)
     [SEE compose-file](https://docs.docker.com/compose/compose-file/)
     """
-    assert settings  # nosec
+    # TODO: should replace this one with the aiodocker version,
+    # can easily get progress out of it and report it back to the UI
```

Comment:
not in the config call. I guess you mean in the pull call?

Reply:
correct